spec/definitions/boxes/definitions/base_ubuntu14.04_kvm/definition.rb

```diff
@@ -0,0 +1,45 @@
+Veewee::Session.declare({
+  :cpu_count => '1',
+  :memory_size => '512',
+  :disk_size => '12040',
+  :disk_format => 'VDI',
+  :hostiocache => 'off',
+  :os_type_id => 'Ubuntu_64',
+  :iso_file => "ubuntu-14.04.3-server-amd64.iso",
+  :iso_src => "http://releases.ubuntu.com/14.04/ubuntu-14.04.3-server-amd64.iso",
+  :iso_md5 => "9e5fecc94b3925bededed0fdca1bd417",
+  :iso_download_timeout => "1000",
+  :boot_wait => "4",
+  :boot_cmd_sequence => [
+    '<Esc><Esc><Enter>',
+    '/install/vmlinuz noapic preseed/url=http://%IP%:%PORT%/preseed.cfg ',
+    'debian-installer=en_US auto locale=en_US kbd-chooser/method=us ',
+    'hostname=base-ubuntu-14.04 ',
+    'fb=false debconf/frontend=noninteractive ',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.